src/Behat/MinkExtension/Listener/SessionsListener.php

```diff
@@ -101,7 +101,12 @@ public function prepareDefaultMinkSession(ScenarioLikeTested $event)
         if ($scenario->hasTag('insulated') || $feature->hasTag('insulated')) {
             $this->mink->stopSessions();
         } else {
-            $this->mink->resetSessions();
+            try {
+                $this->mink->resetSessions();
+            }
+            catch (\Exception $e) {
```

Comment:
It's better to handle `DriverException` or similar Mink exception class. If I'm not mistaken then it would be possible to get actual failed session out of it and stop exactly it.